modules/ossm-installing-operator.adoc

```diff
@@ -5,23 +5,19 @@
 
 .Prerequisites
 
-You have deployed a cluster on {ocp-product-title} 4.13 or later.
+You have deployed a cluster on {ocp-product-title} 4.15 or later.
 
 You are logged in to the {ocp-product-title} web console as a user with the cluster-admin role.
 
-You have access to the OpenShift CLI (oc).
-
 .Procedure
 
 . In the {ocp-product-title} web console, navigate to the *Operators* -> *OperatorHub* page.
 
+ifdef::openshift-rosa[]
 . Scroll, or type the keyword *Serverless* into the *Filter by keyword* box to find the {ServerlessOperatorName}.
```

Comment:
I'm not sure why Serverless shows up here?

Reply:
@dgn It's conditionalized and only appears when serverless doc builds. This comment was in the source file, so I was thinking it was added in the 2.x content and will likely be needed for serverless in 3.0. 

Do you think it should be deleted?

Reply:
Deleted